lib/LANraragi/Plugin/Metadata/RegexParse.pm

```diff
@@ -115,7 +226,7 @@ sub get_tags {
 #(\(([^([)]+)\))? returns the content of (Series). Optional.
 #(\[([^]]+)\])? returns the content of [Language]. Optional.
 #\s* indicates zero or more whitespaces.
-my $regex = qr/(\(([^([]+)\))?\s*(\[([^]]+)\])?\s*([^([]+)\s*(\(([^([)]+)\))?\s*(\[([^]]+)\])?/;
+my $regex = qr/(\(([^([]+)\))?\s*(\[([^]]+)\])?\s*([^([]+)\s*(\(([^([)]+)\))?\s*(\[([^]]+)\])?(?<tail>.*)?/;
```

Comment:
Please document what this extra bit does in the mega-comment above, this regex is really unreadable without it so it needs to stay up to date😅  

From what I can gather this just keeps the remaining trailing part of the filename for the rest of the plugin to execute